### ISSUE TITLE
Azure connection error on macOS using FreeTDS (port issue)

### ISSUE THREAD

**Issue by KimchaC** [mikaelene/dbt-sqlserver#9]
I was having connection problems to Azure.

Using the ODBC driver I was getting this error:
```
Connection:
  database: sm_analysis
  schema: dbt_kim
  UID: hostmaster
  windows_login: False
libc++abi.dylib: terminating with uncaught exception of type std::runtime_error: collate_byname<char>::collate_byname failed to construct for C/en_US.UTF-8/C/C/C/C
[1]    5062 abort      dbt debug
/Users/Kim/.pyenv/versions/3.6.5/lib/python3.6/multiprocessing/semaphore_tracker.py:143: UserWarning: semaphore_tracker: There appear to be 1 leaked semaphores to clean up at shutdown   
  len(cache))
```

Then I tried using FreeTDS, but was getting this error:
```
dbt was unable to connect to the specified database.
The database returned the following error:

  >Database Error
  ('08S01', '[08S01] [FreeTDS][SQL Server]Unable to connect: Adaptive Server is unavailable or does not exist (20009) (SQLDriverConnect)')

Check your database credentials and try again. For more information, visit:
https://docs.getdbt.com/docs/configure-your-profile
```

I finally narrowed down the problem to the fact that the port is not specified in the connection string.

After making these changes I was able to connect:
```
diff --git a/dbt/adapters/sqlserver/connections.py b/dbt/adapters/sqlserver/connections.py
index e336dcc..5a9568c 100644
--- a/dbt/adapters/sqlserver/connections.py
+++ b/dbt/adapters/sqlserver/connections.py
@@ -19,6 +19,7 @@ class SQLServerCredentials(Credentials):
     host: str
     database: str
     schema: str
+    port: Optional[int] = 1433
     UID: Optional[str] = None
     PWD: Optional[str] = None
     windows_login: Optional[bool] = False
@@ -87,6 +88,7 @@ class SQLServerConnectionManager(SQLConnectionManager):
             con_str = []
             con_str.append(f"DRIVER={{{credentials.driver}}}")
             con_str.append(f"SERVER={credentials.host}")
+            con_str.append(f"PORT={credentials.port}")
             con_str.append(f"Database={credentials.database}")
 
             if not getattr(credentials, 'windows_login', False):
```

**Comment by mikaelene** [comment]
Thanks! 

I'll add that asap :-)

**Comment by mikaelene** [comment]
Fixed in branch master, commit 1cf0f061e06f1f17ba22b46c26675ff32dcd00b8 . Thanks for reporting!
